.gitignore

```
@@ -15,7 +15,7 @@ node_modules/
 .npmrc
 **/.cache
 **/.vitepress/cache
-**/.vitepress/config.ts.timestamp-*.mjs
+**/.vitepress/*.ts.timestamp*.mjs
```

```suggestion
*.ts.timestamp*.mjs
```

When starting the docs, I also get another timestamp file which is not inside the .vitepress folder, should we just ignore them all?